app/server.py

```diff
@@ -1,5 +1,6 @@
 import flask
```

Comment:
For one off function usages I think it makes sense to just reference the module.  But once a function is used 3 or 4 times I prefer to just import it separately. So if something is being used as `flask.` and on it's own that's a problem, but I think this system works.